smoketest.sh

```diff
@@ -12,6 +12,8 @@ function runCryostat() {
         GRAFANA_DASHBOARD_URL="http://${host}:${grafanaPort}" \
         CRYOSTAT_RJMX_USER=smoketest \
         CRYOSTAT_RJMX_PASS=smoketest \
+        CRYOSTAT_ALLOW_UNTRUSTED_SSL=true \
+        CRYOSTAT_REPORT_GENERATOR="http://0.0.0.0:10001" \
```

Comment:
What is the difference between `${host}` and `0.0.0.0` here?

Reply:
Good catch. I missed that substitution when I did my rebase. I've fixed it now to use `${host}`